src/trackCopiedText.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * Tracking text copied by the user.
+ * 
+ * Adds event listener
+ */
+
+ export default (subscribe) => {
+    window.addEventListener("copy",function(e){
+        ppas_processCopyEvent(e);
+    });
+    
+    const ppas_processCopyEvent = function(e){
+        let copiedItemText = getSelectedText();
```

Comment:
you shouldn't use a function before it's defined

Reply:
Done